### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KajalSinghBAGHEL**
remove grammatical mistake

### CODE REVIEW

**Comment by KajalSinghBAGHEL** [comment]
I am a GSSOC participant. Please check my pull request

**Comment by Aryamanz29** [comment]
Hii 👋  @KajalSinghBAGHEL 
Thanks for opening PR, but one thing I've noticed that PR doesn't follow the PR template  ( I mean, your PR should be descriptive). Another thing is that did you first open a new issue before raising this PR? I think it should be better if you first open an issue and then raise the PR 😅

File: README.md [hunk 1]
```diff
@@ -21,35 +21,35 @@ Color scheme : https://colorhunt.co/palette/269111
...
+After installation of Django, open this file in any text editor, and in the terminal type the following command to migrate the file. 
 Migrate the files:
 
     python manage.py makemigrations
     python manage.py migrate
 
 And then start the server (default: http://localhost:8000)
 
-    python manage.py runserver
+    python manage.py run server
```
Comment (Aryamanz29) [inline README.md:52]: Please change this to `runserver` because it is a command & It won't work if it contains whitespace.

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

**Comment by sruti2024** [comment]
@Aryamanz29 I do appreciate for your comments as I too have a similar opinion , firstly working on grammatical mistakes and spelling mistakes are not eligible for any marking under gssoc'21 , If you wish to do it just for enhancement then u are heartily welcome !
please keep in mind not to randomly work out an issue if it is not assigned , here I suppose u have not even created one  and please don't change any command unless u have idea of them

**Comment by PROTechThor** [comment]
> @Aryamanz29 I do appreciate for your comments as I too have a similar opinion , firstly working on grammatical mistakes and spelling mistakes are not eligible for any marking under gssoc'21 , If you wish to do it just for enhancement then u are heartily welcome !
> please keep in mind not to randomly work out an issue if it is not assigned , here I suppose u have not even created one and please don't change any command unless u have idea of them

@sruti2024 Eventhough this can't be accepted under GSSoC, can we merge it anyway? There are some good spelling fixes in there.

**Pull request closed without merging**